### ISSUE TITLE
hxCodec issue with Psych Engine 0.6.3

### ISSUE THREAD

**Issue by GamingFan3926** [ShadowMario/FNF-PsychEngine#11311]
### Describe your bug here.

I don't know how to fix this issue! I've updated hxCodec to 2.5.1, I've reinstalled hxCodec god knows how many times, and yet the same damn error keeps popping up. I've tried troubleshooting on this github page. No change. I've tried downloading different versions of hxCodec. Still broken. Any help without the constant repeating of "you're using an old version" is very much appreciated. 

Side note: I tried compiling an older version of Psych (specifically 0.5.1), and the issue isn't there. 

The only issue is with PlayState.hx. Nothing else. For reference, below is the exact file of PlayState.hx I have in my build:

[PlayState.zip](https://github.com/ShadowMario/FNF-PsychEngine/files/10094613/PlayState.zip)

### Command Prompt/Terminal logs (if existing)

```bash
source/PlayState.hx:75: characters 8-22 : Type not found : vlc.MP4Handler
source/PlayState.hx:1620: characters 14-24 : Type not found : MP4Handler
```

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

Yes, I edited Achievements.hx, but I never edited PlayState.hx (which is where the issues are). I want to edit PlayState.hx ONLY when it's working normally.

**Comment by GamingFan3926** [comment]
Update: Figured out the issue. Turns out the Project.xml is the thing that handles the issue for you. I was trying to upgrade my old mod to the latest version (which was on the older version of Psych mentioned above), and dragged over the old Project.xml from that. Turns out this problem could've been avoided if I had just added the new changes I had made for my mod over the new Project.xml :|

TLDR: If you're trying to fix this problem, replace the Project.xml file in your build with the current version's one
